### PR TITLE
feat(app/partner): integrate partner app

### PR DESCRIPTION
**Pull request by LufyCZ**
need to merge in order to set up Vercel and test

### CODE REVIEW

File: packages/ui/app/Header.tsx [hunk 1]
```diff
@@ -25,6 +26,7 @@ const LINK = {
   [AppType.Kashi]: '/internal',
   [AppType.Analytics]: '/analytics',
   [AppType.Pool]: '/pool',
+  [AppType.Partner]: '/partner',
```
Comment (chillichelli) [inline packages/ui/app/Header.tsx:29]: I dont think we should add it to the menu, instead add it to the footer. Its not really a product hence it shouldnt be in the menu

Comment (LufyCZ) [reply packages/ui/app/Header.tsx:29]: it is not in the "active/public" navbar, had to add it here so that it shows up when you're on the page

Comment (chillichelli) [reply packages/ui/app/Header.tsx:29]: Is it not showing now when you're on the swapper and click the menu?

Comment (LufyCZ) [reply packages/ui/app/Header.tsx:29]: it's not no

Comment (chillichelli) [reply packages/ui/app/Header.tsx:29]: Alright cool, lgtm then!

**Review by chillichelli — APPROVED** [review]

**Pull request merged**